### PR TITLE
Toolbar headline

### PR DESCRIPTION
**Pull request by diegoyon**
- Added the toolbar.
- Added the headline section.
- Mobile version only.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <nav>
+      <h2>Welcome</h2>
+      <img src="./images/menu.svg" alt="menu-icon" />
+    </nav>
```
Comment (ShahierNashaat) [inline index.html:20]: - [ ] Kindly be noticed that the tool bar should be in a `header` tag.
- [ ] The logo should be inside a link (`a` tag).
- [ ] The hamburger button should be inside a `nav` tag.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        need coded, don’t hestiate to contact me.
+      </p>
+      <div class="social">
+        <img src="./images/twitter.svg" alt="twitter" />
+        <img src="./images/linkedin.svg" alt="linkedin" />
+        <img src="./images/medium.svg" alt="medium" />
+        <img src="./images/github.svg" alt="github" />
+        <img src="./images/angellist.svg" alt="angellist" />
+      </div>
+      <img src="./images/arrowdown.svg" alt="arrowdown" />
```
Comment (ShahierNashaat) [inline index.html:38]: - [ ] Kindly be noticed that you should wrap all of that section inside a `section` tag.
- [ ] Also be noticed that the list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag).

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,4 @@
+node_modules
+*.svg
+*.jpg
+*.png
```
Comment (ShahierNashaat) [inline .gitignore]: - [ ] Kindly remove the svg, jpg, and png from the .gitignore file as you can push them to the repo so we can view them on the local clone.

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @diegoyon , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (diegoyon) [reply index.html:20]: Quick questions about the logo

- Is the "Welcome"  header the logo?
- Why should it be inside a link? Where should I link it to?

**Review by Ginohmk — APPROVED** [review]
Hi @diegoyon , 

# STATUS: APPROVED 🟢

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 
- You set up linter files and no errors. 💯 

well-1

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**